submissions/damilolaSalami/domManipulation/signUpform.js

```diff
@@ -0,0 +1,55 @@
+const submitform = document.getElementById("validateForm"),
```

Comment:
@queensaldam It will be good practice to have variable names in camelCase for readability, you should update them as well as the IDs too.